### PR TITLE
Create staedlma-Berlin.txt

### PR DESCRIPTION
**Pull request by Staedlma**
created new file

### CODE REVIEW

File: staedlma-Berlin.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Places to eat
+- FrauMittenmang
```
Comment (githubteacher) [inline staedlma-Berlin.txt:2]: Really like white sausage, any recommendations on places to go?
Also, really like German lagers, where is your favorite place to grab a liter?

**Comment by githubteacher** [comment]
Can you add some places to visit while in Berlin?

**Review by hectorsector — APPROVED** [review]
Go ahead and 🚢!

File: staedlma-Berlin.txt (hunk 1 above)
Comment (Staedlma) [reply staedlma-Berlin.txt:2]: they have self made german lager, very good!

**Pull request merged**